autonomy/deploy/generators/docker_compose/base.py

```diff
@@ -74,6 +76,18 @@
 DEFAULT_OPEN_AUTONOMY_DIR: Path = Path.home().absolute() / "open-autonomy"
 
 
+def get_docker_client() -> DockerClient:
+    """Load docker client."""
+    try:
+        return from_env()
+    except DockerException:
+        return DockerClient(
+            DEFAULT_NPIPE
+            if IS_WINDOWS_PLATFORM
+            else f"unix://{Path.home()}/.docker/desktop/docker.sock"
+        )
```

Comment:
This incorporates the fix user suggested here - https://github.com/valory-xyz/trader-quickstart/issues/251